src/functions/delimsizing.js

```diff
@@ -161,7 +159,8 @@ defineFunction({
     },
     htmlBuilder: (group, options) => {
         // Build the inner expression
-        const inner = html.buildExpression(group.value.body, options, true);
+        const inner = html.buildExpression(group.value.body, options, true,
+            [null, "mclose"]);
```

Comment:
This seems to be the on place where the `surrounding` param is used.  Do you for other situations where we'll need the first value instead of just the second?

Reply:
@kevinbarabash Not yet, but I think it makes sense to have for both sides.